test/plain-list/AutosuggestApp.test.js

```diff
@@ -456,6 +803,19 @@ describe('Plain list Autosuggest', () => {
       });
     });
 
+    it('should not call onChange when Enter is pressed and suggestion is focused with down since value did not change', () => {
+      onChange.reset();
+      clickDown();
+      clickEnter();
+
+      // onChange should not have been called because the value didn't change
+      expect(onChange).to.have.been.calledExactlyOnce;
+      expect(onChange).to.be.calledWithExactly(eventInstance, {
+        newValue: 'Java',
+        method: 'down'
+      });
+    });
```

Comment:
Ok, I updated the logic here to `onChange.clear` after the Down event, so it should be more clear that we're testing to make sure there's no `onChange` for only the `Enter` event.

See https://github.com/moroshko/react-autosuggest/pull/165/commits/19e5409f5f05cc5f0f633d967ef3c544dfec22cb